app/(auth)/signup/future/page.tsx

```diff
@@ -1,13 +1,12 @@
 /* eslint-disable @typescript-eslint/no-misused-promises */
```

Comment:
@djsiddz as a best practise always try to handle the lint rule than disabling it. If you cannot handle it then put the disable comment on that line and not globally.

Reply:
Making changes for these.